applications/rpg_table_helper/build.yaml

```
@@ -20,4 +24,16 @@
+
+      chopper_generator:
+        options:
+          header: "// Generated code"
+      swagger_dart_code_generator:
+        options:
+          input_folder: "swagger/"
+          output_folder: "lib/generated/swaggen/"
+          with_base_url: true
+          enums_case_sensitive: false
+          separate_models: true
+          
```

yamllint

[error] trailing spaces

(trailing-spaces)